packages/types/src/promotion/common/campaign-budget.ts

```diff
@@ -0,0 +1,16 @@
+import { BaseFilterable } from "../../dal"
+
+export type CampaignBudgetTypeValues = "spend" | "usage"
```

Comment:
Question: Is there a reason not to use an enum here when you've used an enum in types and the model? 

also renaming suggestion:
```suggestion
export type CampaignBudgetType = "spend" | "usage"
```

Reply:
enums can't be imported into the types package if we want to keep this purely a dev dependency.

So I've added the enum in the utils package (`packages/utils/src/promotion/index.ts` in the same file) and added the type of the values here.

I want to keep this `CampaignBudgetTypeValues` in order to not mix it up with the enum `CampaignBudgetType`